src/main/kotlin/se/uulm/snowballr/backend/validation/ProjectValidator.kt

```diff
@@ -67,6 +67,20 @@ object ProjectValidator {
         ) { _, _, _, _, _ -> }
     }
 
+    fun validateGetInformationRequest(request: Project.Information.Get): EitherNel<ValidationIssue, Unit> = either {
+        // Validate the field mask
+        val fieldMaskResult = either {
+            ensureFieldMaskIsValid(request.mask, Project.Information.Get.getDescriptor())
+        }
+
+        // If field mask validation fails, return early
+        if (fieldMaskResult is Either.Left) {
+            fieldMaskResult.toEitherNel().bind()
+        }
+
+        either { ensureIdValidity("id", request.projectId) }.toEitherNel().bind()
```

Comment:
The field name is `project_id`.
```suggestion
        either { ensureIdValidity("project_id", request.projectId) }.toEitherNel().bind()
```